Upstream-Status: Inappropriate [embedded specific]

Index: seabios-1.17.0/Makefile
===================================================================
--- seabios-1.17.0.orig/Makefile
+++ seabios-1.17.0/Makefile
@@ -22,7 +22,7 @@ LD=$(CROSS_PREFIX)ld
 OBJCOPY=$(CROSS_PREFIX)objcopy
 OBJDUMP=$(CROSS_PREFIX)objdump
 STRIP=$(CROSS_PREFIX)strip
-PYTHON=python
+PYTHON=python3
 CPP=$(CROSS_PREFIX)cpp
 IASL:=iasl
 LD32BIT_FLAG:=-melf_i386
